openapi/openapi_V1.yaml

```diff
@@ -0,0 +1,46 @@
+swagger: '2.0'
+info:
+  title: Microservice JAVA API
+  description: Microservice JAVA API
+  version: '0.0.1'
+host: localhost
+basePath: /api/v1/payment
+schemes:
+  - https
+
+tags:
+  - name: API  Payment
+    description: API  Payment
+
+paths:
+  /check/{noticeNumber}:
```

Comment:
How is possible to use the noticeNumber as key? The notice number is not unique.
I would expect to have the message id.